### PR TITLE
magento/devdocs#: Declarative schema and Develop data and schema patches. Execution priority.

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) provides information about the execution priority of `Declarative schema` and `Develop data and schema patches`.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.html
- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/data-patches.html

## Links to Magento source code

- [Magento\Setup\Model\Installer::installSchema](https://github.com/magento/magento2/blob/2.3-develop/setup/src/Magento/Setup/Model/Installer.php#L835)

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ The declarative schema approach removes the version from the `setup_module` tabl
 
 The sequence of installing patches is handled through a dependency-based approach. Patches can either be independent or dependent on other patches. Independent patches can be installed in any sequence. A dependent patch requires a minimal number of patches so that it can be installed successfully.
 
+{: .bs-callout-info }
+Magento prioritize a declarative schema approach and executes updates from the [db_schema.xml]({{ page.baseurl }}/extension-dev-guide/declarative-schema/db-schema.html) before the data and schema patches.
```
Comment (rogyar) [inline guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md]:
```suggestion
Magento prioritizes the declarative schema approach and executes updates from the [db_schema.xml]({{ page.baseurl }}/extension-dev-guide/declarative-schema/db-schema.html) before the data and schema patches.
```

File: guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ The main disadvantage of this approach is that Magento applies changes blindly.
 Declarative setup is based on database structure declarations, and is used in projects such as [Doctrine](http://www.doctrine-project.org/). Schema files declare what the database structure should be,
 and Magento determines the differences between the current table structure and what it should be. These differences can be represented with atomic SQL operations.
 
+Magento prioritize declarative schema and executes declarative install schemas before the [data and schema patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html).
```
Comment (rogyar) [inline guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md]:
```suggestion
Magento prioritizes declarative schema and executes the declarative schemas processing before the [data and schema patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html).
```

Comment (rogyar) [reply guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md]: Could you also review the references (links) you provide in the description? I believe we can use the same set of references for both documents. Currently, you have a reference (link) in the `data-patches.md` that is absent in `db-schema.md` and vice versa: 

[data and schema patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html)
[db_schema.xml]({{ page.baseurl }}/extension-dev-guide/declarative-schema/db-schema.html)

Thank you!

File: guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md (hunk 1 above)
Comment (atwixfirster) [reply guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md]: **fixed**

File: guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md]: **fixed**

Thank you, @rogyar !

**Pull request merged**